schemas/bursar-export/bursarExportJob.json

```diff
@@ -0,0 +1,52 @@
+{
+  "description": "Bursar export job schema",
+  "type": "object",
+  "properties": {
+    "filter": {
+      "description": "Filter to use for the regular export",
+      "$ref": "filter/bursarExportFilter.json"
+    },
+    "groupByPatron": {
+      "description": "Choose whether to group fee/fines by patron",
+      "type": "boolean"
+    },
+    "groupByPatronFilter": {
+      "description": "Filter to use for the export when grouping by patron",
+      "$ref": "filter/bursarExportFilterAggregate.json"
+    },
+    "header": {
+      "description": "Header format for the export file",
+      "type": "array",
+      "items": {
+        "$ref": "tokens/bursarExportHeaderFooter.json"
```

Comment:
as on UI, IMO it's a bad name when you place 2 things like HeaderFooter

Reply:
correct me if I'm wrong
header and footer and actually headerTokens/Formats/Whatewer and it contains array of different kinds tokens (conditional, date, etc)

if it's true, mb it makes sense to make clear names?

-----

what I read from the current schema: there is a job with header/footer that contains array of smth HeaderFooter

Reply:
IMO the `tokens/` directory is sufficient

Reply:
do we read directories working with API? @ncovercash naming should be improved, it affects both FE and BE

Reply:
what's the problem to call headerTokens/exportHeaderTokens? and across repos code will be more readable and clear

and bursarExportHeaderFooter.json can be just bursarTokens.json, not smth headerFooter